Core/Consent String Specification.md

```diff
@@ -173,18 +173,33 @@ To decode a code word, remove the final "1", assign the remaining values 1,2,3,5
 
 The following details provide information on creating, storing, and managing a GPP String. The basic steps for creating a GPP String are as follows: 
 
-1. **Create discrete sections.** For each section: 
- - For sections that use the recommended base64-websafe encoding, create a bit representation of the section’s header (if it exists) and each sub-section adding padding (0) on the right to get to a multiple of 6. Then, convert them to base64-websafe without padding (i.e. removing “=” at the end) and concatenate the header and all sub-sections using the “.” (dot) character.
- - For sections that use different encoding, ensure that the data is websafe and does not include the “~” (tilde) character.
-2. **Create header section.** Create a bit representation of the GPP header section. Include all IDs for discrete sections in a sorted order. Add padding (0) on the right to get to a multiple of 6. Then, convert it to base64-websafe without padding.
-3. **Concatenate all sections.** Concatenate the GPP-header as the first item to the encoded versions of the discrete sections using “~” (tilde) as the delimiter.
-
-
-# How should the GPP String be stored?
+<ol>
+<li><b>Create discrete sections.</b>
+<ul><li>For sections that use the recommended encoding mechanism:
+	<ol type=1>
+		<li>Create a bit representation of the section's header (if it exists).</li>
+		<li>Add padding (0) on the right to get to a multiple of 6.</li>
```

Comment:
Perhaps `<li>Add padding (0) on the right to get to a total bit length that is a multiple of 6.</li>`